src/filters/fs.rs

```diff
@@ -63,6 +63,13 @@ pub fn file(path: impl Into<PathBuf>) -> impl FilterClone<Extract = One<File>, E
         })
 }
 
+
+/// useful if file path needs to be looked up in database etc.
+/// use conditionals() filter to get a Conditionals, then reply with this function
+pub async fn serve_file_reply(path: impl Into<PathBuf>, conditionals: Conditionals) -> Result<File, Rejection> {
```

Comment:
This could be useful, but I'd rather discuss the proper API in a new issue, yea?

Reply:
Absolutely, since I'm used to gerrit I didn't realize that this would end up in the same PR :/

Reply:
Done on karlri:concat-performance